mobile/src/home-view.js

```diff
@@ -443,8 +443,8 @@ class HomeView extends PureComponent {
     const { sessions, sessionId } = this.state
     const session = sessions[sessionId]
     const { question } = session
-    const title = question.text.replace('.',',').replace('#','hashtag').replace('$','USD').replace('[','|').replace('}','`|')
-    this.answersRef().update({ [sessionId]: i })
+    const title = question.text.replace(/./g,',').replace(/#/g,'hashtag').replace(/$/g,'USD').replace('[','|').replace('}','`|')
```

Comment:
This feels like a hacky way to avoid disallowed characters in a key.  Why don't we just not put user-generated text in keys?